### PR TITLE
accept null standard error (buffer | string | null)

### PR DESCRIPTION
**Pull request by joaquimsantmrf**
We saw a case in production where "cannot read length of null" error is given as a result of validation of an extensible json ((https://bobthebuilder.marfeel.com:8443/view/3.ProTenants/job/ProTenants/20083/console)).
Seems stderr appears to be null, we add this case. According to documentation should return buffer o r string... In local the mrf-validate-json ALL from the mediagroup folder validates properly, also mrf-json //file//path  is working...

https://nodejs.org/api/child_process.html#child_process_child_process_spawnsync_command_args_options

### CODE REVIEW

**Comment by joaquimsantmrf** [comment]
retest this please

**Comment by joaquimsantmrf** [comment]
retest this please

File: json-lint/utils/schemaUtils.js [hunk 1]
```diff
@@ -25,7 +25,7 @@ const getErrorMessage = (command, jsonPath, status, execPath) => `Error merging
 function getMarfeelExtendedJson(jsonPath, command = JSON_MERGE_COMMAND ) {
 	const mrfJson = spawnSync(command , [ jsonPath ]);
 	
-	if(mrfJson.stderr.length > 0 ||!!mrfJson.status){
+	if((mrfJson.stderr && mrfJson.stderr.length > 0) ||!!mrfJson.status){
```
Comment (lucacamplimrf) [inline json-lint/utils/schemaUtils.js]: const (and clean code) are for free :P
```suggestion
        const hasErrors = mrfJson.stderr && mrfJson.stderr.length > 0;
	if(hasErrors || mrfJson.status !== 0) {
```

**Comment by joaquimsantmrf** [comment]
retest this please

**Comment by joaquimsantmrf** [comment]
retest this please

**Pull request closed without merging**